cclib/parser/orcaparser.py

```diff
@@ -2015,21 +2015,17 @@ def energy_intensity(line):
             if "cpu_time" not in self.metadata:
                 self.metadata['cpu_time'] = []
             
-            self.metadata['wall_time'].append(datetime.timedelta(
+            dtime = datetime.timedelta(
                 days = days,
                 hours = hours,
                 minutes = minutes,
                 seconds = seconds,
                 milliseconds = milliseconds
-            ))
+            )
+            dtime = f"{dtime}"
```

Comment:
is this what is fixing the dump to json? Before we are storing the actual `timedelta` object. If this is what is fixing it, we should fix the json at the dumping point to change `metadata['wall_time']` to str before dumping to json. This would currently be an issue in other parsers as well

Reply:
Also, `str(dtime)` works just as well.